mobile/lib/features/wallet/balance_change_notifier.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:flutter/material.dart';
+import 'package:get_10101/bridge_generated/bridge_definitions.dart';
+
+class BalanceChangeNotifier extends ChangeNotifier {
```

Comment:
Note: This will likely result in conflicts with https://github.com/get10101/10101/pull/99 but should not be too hard to resolve